Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1540,6 +1540,51 @@ public void TestInvalidProxySettingFromConnectionString()
             }
         }
 
+        [Test]
+        [TestCase("*")]
+        [TestCase("*{0}*")]
+        [TestCase("^*{0}*")]
+        [TestCase("*{0}*$")]
+        [TestCase("^*{0}*$")]
+        [TestCase("^nonmatch*{0}$|*")]
+        [TestCase("*a*", "a")]
+        [TestCase("*la*", "la")]
+        public void TestNonProxyHostShouldBypassProxyServer(string regexHost, string proxyHost = null)
+        {
+            using (var conn = new SnowflakeDbConnection())
+            {
+                var host = ResolveHostToUseFromTestConfig(ConnectionString);
+                var nonProxyHosts = string.Format(regexHost, $"{host}");
+                var proxyHostForConnection = proxyHost ?? "proxyserverhost";
```

Comment:
I would rather pass "proxyserverhost" directly as proxyHost without default null value. Maybe it would be a little code duplication, but the test would be a little bit more readable.

Reply:
I change the default value of the method parameter to use "proxyserverhost" instead of null. Thank you!!